include/deal.II/fe/fe_series.h

```diff
@@ -96,24 +96,29 @@ namespace FESeries
      * each direction, @p fe_collection is the hp::FECollection for which
      * expansion will be used and @p q_collection is the hp::QCollection used to
      * integrate the expansion for each FiniteElement in @p fe_collection.
+     * @p component defines which component of each FiniteElement will be used,
+     * it is mainly used for non-primitive elements.
      */
     Fourier(const std::vector<unsigned int> &      n_coefficients_per_direction,
             const hp::FECollection<dim, spacedim> &fe_collection,
-            const hp::QCollection<dim> &           q_collection);
+            const hp::QCollection<dim> &           q_collection,
+            const unsigned int                     component = 0);
 
     /**
      * A non-default constructor. The @p n_coefficients_per_direction defines the
      * number of modes in each direction, @p fe_collection is the hp::FECollection
      * for which expansion will be used and @p q_collection is the hp::QCollection
      * used to integrate the expansion for each FiniteElement
-     * in @p fe_collection.
+     * in @p fe_collection. @p component defines which component of each
+     * FiniteElement will be used, it is mainly used for non-primitive elements.
      *
      * @deprecated Use a different constructor instead.
      */
     DEAL_II_DEPRECATED
     Fourier(const unsigned int                     n_coefficients_per_direction,
             const hp::FECollection<dim, spacedim> &fe_collection,
-            const hp::QCollection<dim> &           q_collection);
+            const hp::QCollection<dim> &           q_collection,
+            const unsigned int                     component = 0);
```

Comment:
This constructor is deprecated anyway, you probably don't need to add the `component` argument.